cfn/deploy_ecs.yaml

```diff
@@ -0,0 +1,329 @@
+        Value: dle
+      - Key: com.docker.compose.network
+        Value: dle_default
+      VpcId: !Ref VPCParameter
+    Type: AWS::EC2::SecurityGroup
+  DefaultNetworkIngress:
+    Properties:
+      Description: Allow communication within network default
+      GroupId:
+        Ref: DefaultNetwork
+      IpProtocol: "-1"
+      SourceSecurityGroupId:
+        Ref: DefaultNetwork
+    Type: AWS::EC2::SecurityGroupIngress
+  DjangoService:
+    DependsOn:
+    - DjangoTCP8000Listener
+    Properties:
+      ServiceName: !Ref ecsServiceName
+      EnableExecuteCommand: True
+      Cluster:
+        Fn::GetAtt:
+        - Cluster
+        - Arn
+      DeploymentConfiguration:
+        MaximumPercent: 200
+        MinimumHealthyPercent: 100
+      DeploymentController:
+        Type: ECS
+      DesiredCount: 1
+      LaunchType: FARGATE
+      LoadBalancers:
+      - ContainerName: django
+        ContainerPort: 8000
+        TargetGroupArn:
+          Ref: DjangoTCP8000TargetGroup
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: ENABLED
+          SecurityGroups:
+          - Ref: DefaultNetwork
+          Subnets:
+          - !Ref subnet1
+      PlatformVersion: 1.4.0
+      PropagateTags: SERVICE
+      SchedulingStrategy: REPLICA
+      ServiceRegistries:
+      - RegistryArn:
+          Fn::GetAtt:
+          - DjangoServiceDiscoveryEntry
+          - Arn
+      Tags:
+      - Key: com.docker.compose.project
+        Value: dle
+      - Key: com.docker.compose.service
+        Value: django
+      TaskDefinition:
+        Ref: DjangoTaskDefinition
+    Type: AWS::ECS::Service
+  DjangoServiceDiscoveryEntry:
+    Properties:
+      Description: '"django" service discovery entry in Cloud Map'
+      DnsConfig:
+        DnsRecords:
+        - TTL: 60
+          Type: A
+        RoutingPolicy: MULTIVALUE
+      HealthCheckCustomConfig:
+        FailureThreshold: 1
+      Name: django
+      NamespaceId:
+        Ref: CloudMap
+    Type: AWS::ServiceDiscovery::Service
+  DjangoTCP8000Listener:
+    Properties:
+      DefaultActions:
+      - ForwardConfig:
+          TargetGroups:
+          - TargetGroupArn:
+              Ref: DjangoTCP8000TargetGroup
+        Type: forward
+      LoadBalancerArn:
+        Ref: loadBalancerArn
+      Port: 8000
+      Protocol: TCP
+    Type: AWS::ElasticLoadBalancingV2::Listener
+  DjangoTCP8000TargetGroup:
+    Properties:
+      Port: 8000
+      Protocol: TCP
+      Tags:
+      - Key: com.docker.compose.project
+        Value: dle
+      TargetType: ip
+      VpcId: !Ref VPCParameter
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+  DjangoTaskDefinition:
+    Properties:
+      EphemeralStorage:
+        SizeInGiB: 40
+      TaskRoleArn: !Ref DjangoTaskExecutionRole
+      ContainerDefinitions:
+      - Command:
+        - us-east-1.compute.internal
+        - dle.local
+        Essential: false
+        Image: docker/ecs-searchdomain-sidecar:1.0
```

Comment:
This is the built image we push to ECR? Just curious, a bit confused by the name

Reply:
no, the image is defined in the SSM param. this is for the init container